packages/react-redux-subspace/package.json

```diff
@@ -31,47 +31,49 @@
     "url": "https://github.com/ioof-holdings/redux-subspace.git"
   },
   "dependencies": {
-    "@types/react": "^16.4.18",
-    "hoist-non-react-statics": "^3.0.1",
-    "prop-types": "^15.6.0",
+    "@types/react": "^16.8.3",
+    "hoist-non-react-statics": "^3.3.0",
+    "memoize-one": "^5.0.0",
+    "prop-types": "^15.7.2",
     "recompose": "^0.30.0",
     "redux-subspace": "^3.0.1"
   },
   "peerDependencies": {
     "react": "^15.1 || ^16.0",
-    "react-redux": "^5.0.0",
+    "react-redux": "^5.0.0 || ^6.0.0",
```

Comment:
current implementation should be backwards compatible, but it has complicated the internals.  We could do a clean cut over and do a major bump instead.  I'm just worried about how we currently distribute `subspace` and `dynostore` through our bundles, not having backwards compatibility will make upgrading harder.

Reply:
Yeah we will definately want to have this version supporting both.

Even if we release the major bump soon after just to cleanup and prepare for a "no legacy context" future, well probably be running this version for a while until we can port all the micro-frontends over.

Reply:
Talking internal to ioof now. Because of the new context api and having to be the same reference, we're going to externalize subspace (and dynostore?), Upgrade all the micro-frontends so they are using the same version from the parent, then upgrade react-redux.

We actually won't need to support both contexts at once? If we first externalize the old version and then cut over to the new.

We may still want to have this version for others to migrate with.

Reply:
Because the context we use comes from `react-redux` which is already externalised, we don't _have_ to externalise subspace or dynostore for this change...

But I **_really, really_** think we should so that future changes like this are easier.  Anything where there is a provider in one bundle and a consumer in another should, in my opinion, be externalised.